kafka/conn.py

```diff
@@ -220,6 +221,15 @@ def _wrap_ssl(self):
                 self._ssl_context.load_cert_chain(
                     certfile=self.config['ssl_certfile'],
                     keyfile=self.config['ssl_keyfile'])
+            if self.config['ssl_crlfile']:
+                if not hasattr(ssl, 'VERIFY_CRL_CHECK_LEAF'):
+                    log.error('No CRL support with this version of Python.'
+                              ' Disconnecting.', self)
```

Comment:
Ok, this is a pylint error. I corrected the error. Let's see if this works with Travis.